### DISCUSSION TITLE
Why are there 2 version of the kratos client?

### DISCUSSION THREAD

**Discussion by mwahlhuetter** [ory/kratos#2336] [Q&A] [answered]
There are two clients @ory/client and @ory/kratos-client in the npm registry.
What is the difference and which one should be used? I could not find  something about this in the docs.

**Comment by vinckr** [comment, marked as answer]
Hey @mwahlhuetter 

We have recently restructured our clients, the correct one is 
[`@ory/client`.](https://www.npmjs.com/package/@ory/client)

It will be more structured and less confusing soon 🙏, we are basically moving all individual packages to a shared "ory/client"

edit: I will mark this as answered for now, feel free to reply if something is unclear still.

**Reply by mwahlhuetter** [reply]
That clears everything up.
Thank you for your quick response!
